riscv/mmu.h

```diff
@@ -225,6 +225,18 @@ class mmu_t
     return (addr / PC_ALIGN) % ICACHE_ENTRIES;
   }
 
+  inline reg_t fetch_jump_table(reg_t addr, int length)
+  {
+    auto tlb_entry = translate_insn_addr(addr, true);
```

Comment:
I think all of this will change soon; there is discussion about making fetch table accesses perform read permission checks instead of fetch permission checks.  I recommend we not merge this until that change has been made, because it will dramatically change this code.

After that change is made, I think you can simply call `load_slow_path_intrapage` here.  I don't want to create special cases for `is_jvt`.

Reply:
OK. I'll update this if spec changes.

Reply:
BTW, if the change is not made, and we instead continue to use execute permissions, then I have an alternate suggestion.

The spec says "It is recommended that the second fetch be ignored for hardware triggers and breakpoints."  That is, it is still legal to check the triggers.  So, I recommend that we take advantage of this property, and continue to check the triggers, to avoid adding a special case to this complicated code.